src/includes/getting-started-install/java.mdx

```diff
@@ -1,4 +1,4 @@
-To install the Android SDK, add it your `build.gradle` file:
+To install the SDK, add it your `build.gradle` file:
```

Comment:
This install is rather bias towards Android usage (assumes Gradle.)

Under "usage" we have install instructions with gradle plus other options:

https://sentry-docs-git-update-java-getting-started.sentry.dev/platforms/java/usage/#installation

Ideally that would be merged with this (that gradle instruction doesn't suggest adding a repository so could take the approach used here instead.

Reply:
Yes, I saw those more detailed instructions and wondered if they might be a good source. But we can't make it confusing by having too many options. For JS, for example, we document how to install with yarn or npm on the main Getting Started page, but we pulled the CDN instructions and lazy loading into separate pages.

So, I think our best plan is to tell developers what we anticipate is the most used form(s), with a link to other install methods. Do you think that would work here?